apps/minifront/src/state/helpers.ts

```diff
@@ -26,49 +79,32 @@ export const plan = async (
   return plan;
 };
 
-export const authWitnessBuild = async (
```

Comment:
Combined `authWitnessBuild` and `witnessBuild`, since they were identical other than the `viewClient` method they called.